framework/src/schema/application_config_schema.ts

```diff
@@ -12,10 +12,94 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
+
+const moduleConfigSchema = {
+	type: 'object',
+	propertyNames: {
+		pattern: '^[a-zA-Z][a-zA-Z0-9_]*$',
+	},
+	additionalProperties: { type: 'object' },
+};
+
+const rewardsConfigSchema = {
+	type: 'object',
+	required: ['milestones', 'offset', 'distance'],
+	description: 'Object representing LSK rewards milestone',
+	properties: {
+		milestones: {
```

Comment:
the naming has changed, maybe reflect the change? https://github.com/LiskHQ/lips-staging/pull/94/files#diff-dfeda675218ab1dec2be752051c6b2a96bd201a0d704ebbb438c31b32312fb25R63
@IkerAlus its `brackets` now?

Reply:
When the implementation of that LIP starts, we should make that change.